sample_info/modules/ntk.py

```diff
@@ -11,7 +11,16 @@
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 
+"""
+TODO: write description and notes.
```

Comment:
Take care of this `TODO`?

Reply:
Added a single-sentence description of `ntk.py`.